core/src/services/sftp/backend.rs

```diff
@@ -104,6 +105,7 @@ pub struct SftpBuilder {
     user: Option<String>,
     key: Option<String>,
     known_hosts_strategy: Option<String>,
+    copyable: bool,
```

Comment:
I prefer to use `<verb>_<op>` like `can_copy` or `enable_copy`. In this way, we don't need to play with english grammer like `movable` (we can't use `moveable`), `presignable` (there is no such word).